raiden/tests/integration/network/test_pathfinding.py

```diff
@@ -162,9 +162,21 @@ def test_configure_pfs(service_registry_address, private_keys, web3, contract_ma
                 node_network_id=ChainID(chain_id),
                 token_network_registry_address=token_network_registry_address_test_default,
                 pathfinding_max_fee=DEFAULT_PATHFINDING_MAX_FEE,
-                matrix_servers=["http://matrix.doesnotexist.com"],
+                matrix_servers=["matrix.doesnotexist.com"],
             )
 
+    # Correct matrix server with just the protocol added should pass
+    with patch.object(requests, "get", return_value=response):
+        configure_pfs_or_exit(
+            pfs_url="http://foo",
+            routing_mode=RoutingMode.PFS,
+            service_registry=service_registry,
+            node_network_id=ChainID(chain_id),
+            token_network_registry_address=token_network_registry_address_test_default,
+            pathfinding_max_fee=DEFAULT_PATHFINDING_MAX_FEE,
+            matrix_servers=["matrix.example.com"],
+        )
+
```

Comment:
This extra test is unnecessary if you remove the `http://` from the server name in the tests above.

Reply:
Good point